<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="widget" native="true">
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="4">
         <widget class="QToolButton" name="buttonExport">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="icon">
           <iconset resource="resources/resources.qrc">
            <normaloff>:/images/actions/document-export.svg</normaloff>:/images/actions/document-export.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">projectButtons</string>
          </attribute>
         </widget>
        </item>
        <item row="0" column="1">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Preferred</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>22</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="5">
         <widget class="QToolButton" name="buttonTools">
          <property name="icon">
           <iconset resource="resources/resources.qrc">
            <normaloff>:/images/actions/run-build-configure.svg</normaloff>:/images/actions/run-build-configure.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QToolButton" name="buttonMainMenu">
          <property name="icon">
           <iconset resource="resources/resources.qrc">
            <normaloff>:/images/actions/application-menu.svg</normaloff>:/images/actions/application-menu.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
          <property name="popupMode">
           <enum>QToolButton::InstantPopup</enum>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QToolButton" name="buttonFindAll">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="icon">
           <iconset resource="resources/resources.qrc">
            <normaloff>:/images/actions/edit-find-replace.svg</normaloff>:/images/actions/edit-find-replace.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">projectButtons</string>
          </attribute>
         </widget>
        </item>
        <item row="1" column="0" colspan="6">
         <widget class="QTreeView" name="fileViewer">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>1</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="dragEnabled">
           <bool>true</bool>
          </property>
          <property name="dragDropMode">
           <enum>QAbstractItemView::DragDrop</enum>
          </property>
          <property name="defaultDropAction">
           <enum>Qt::TargetMoveAction</enum>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
          <property name="textElideMode">
           <enum>Qt::ElideMiddle</enum>
          </property>
          <property name="uniformRowHeights">
           <bool>true</bool>
          </property>
          <property name="animated">
           <bool>true</bool>
          </property>
          <property name="headerHidden">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QToolButton" name="buttonReload">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="icon">
           <iconset resource="resources/resources.qrc">
            <normaloff>:/images/actions/view-refresh.svg</normaloff>:/images/actions/view-refresh.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">projectButtons</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QListView" name="actionViewer">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QComboBox {
    border: 1px solid #cccccc;
    border-radius: 8px;
    padding: 1px 18px 1px 13px;
    min-width: 6em;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 15px;

    border-left-width: 0px;
    border-left-color: darkgray;
    border-left-style: solid;
    border-top-right-radius: 3px;
    border-bottom-right-radius: 3px;
}

QComboBox QAbstractItemView {
    border: 1px solid black;
    padding-left: 13px;
}
</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QComboBox" name="documentSelector">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>135</width>
           <height>29</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QToolButton" name="buttonDocumentSave">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="icon">
          <iconset resource="resources/resources.qrc">
           <normaloff>:/images/actions/document-save.svg</normaloff>:/images/actions/document-save.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>22</width>
           <height>22</height>
          </size>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">editionButtons</string>
         </attribute>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QToolButton" name="buttonDocumentSaveAll">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="icon">
          <iconset resource="resources/resources.qrc">
           <normaloff>:/images/actions/document-save-all.svg</normaloff>:/images/actions/document-save-all.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>22</width>
           <height>22</height>
          </size>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">editionButtons</string>
         </attribute>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QToolButton" name="buttonDocumentClose">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="icon">
          <iconset resource="resources/resources.qrc">
           <normaloff>:/images/actions/document-close.svg</normaloff>:/images/actions/document-close.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>22</width>
           <height>22</height>
          </size>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">editionButtons</string>
         </attribute>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QToolButton" name="buttonDocumentCloseAll">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="icon">
          <iconset resource="resources/resources.qrc">
           <normaloff>:/images/actions/document-close-all.svg</normaloff>:/images/actions/document-close-all.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>22</width>
           <height>22</height>
          </size>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">editionButtons</string>
         </attribute>
        </widget>
       </item>
       <item row="1" column="0" colspan="5">
        <widget class="QSplitter" name="splitter_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="DocumentManager" name="documentContainer" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>1</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="sizeIncrement">
           <size>
            <width>0</width>
            <height>600</height>
           </size>
          </property>
         </widget>
         <widget class="QTextBrowser" name="logView">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
      <zorder>buttonDocumentCloseAll</zorder>
      <zorder>buttonDocumentClose</zorder>
      <zorder>buttonDocumentSaveAll</zorder>
      <zorder>buttonDocumentSave</zorder>
      <zorder>documentSelector</zorder>
      <zorder>splitter_2</zorder>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DocumentManager</class>
   <extends>QWidget</extends>
   <header location="global">documentmanager.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources/resources.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="projectButtons"/>
  <buttongroup name="editionButtons"/>
 </buttongroups>
</ui>
